client/state/selectors/has-initialized-sites.js

```diff
@@ -1,10 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import 'calypso/state/ui/init';
+
 /**
  * Returns true if site selection has occured, else false
  *
  * @param {object}  state Global state tree
  * @returns {boolean}       Has site selection occurred
  */
-
 export default function hasInitializedSites( state ) {
 	return state.ui.siteSelectionInitialized;
```

Comment:
Since we expect a `boolean` here, is it concerning we're not doing something like `state.ui?.siteSelectionInitialized` or `state.ui?.siteSelectionInitialized ?? false`?